shell/platform/windows/window_state.h

```diff
@@ -39,13 +39,24 @@ struct FlutterDesktopView {
   flutter::Win32FlutterWindow* window;
 };
 
+struct AotDataDeleter {
+  void operator()(FlutterEngineAOTData aot_data) {
+    FlutterEngineCollectAOTData(aot_data);
```

Comment:
Nit: Not sure if null aot_data's can be collected. Maybe a paranoid check here?

Reply:
FlutterEngineCollectAOTData has an internal null check.